src/client/app/shared/prescription/prescription.component.ts

```diff
@@ -0,0 +1,393 @@
+import { Component, OnInit, ChangeDetectionStrategy, ChangeDetectorRef, Input, Output, ViewChild, ElementRef } from '@angular/core';
+import { FormBuilder, FormArray, FormGroup, FormControl, Validators  } from '@angular/forms';
+import * as jsPDF from 'jspdf';
+import 'jspdf-autotable';
+import { EventEmitter } from '@angular/core';
+import { ChatService } from '../../chat/chat.service';
+import { ProfileService } from '../../profile/profile.service';
+
+@Component({
+    moduleId: module.id,
+    selector:'mm-prescription',
+    templateUrl:'prescription.component.html',
+    styleUrls:['style.css'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+
+})
+
+export class ElectronicPrescriptionComponent implements OnInit {
```

Comment:
Naming convention violation. File name is `prescription.component.ts` and the exported class is `ElectronicPrescriptionComponent`. Change it to `PrescriptionComponent`.